### ISSUE TITLE
Some MIDIs do not loop

### ISSUE THREAD

**Issue by afritz1** [afritz1/OpenTESArena#29]
The MIDI track SHEET.XMI loops over and over, but SUNNYDAY.XMI (in the game world) only plays once.

**Comment by kcat** [comment]
This seems to be a problem with the song, or at least how it's interpreted by WildMIDI. I remember having the same issue with some of Daggerfall's songs too (some of which are the same as Arena's, but in HMI format). Ultimately the problem is that the song has a really really really large delay specified before the end of the song (SUNNYDAY.XMI is calculated to be over 166 minutes long).

Probably what needs to happen is that WildMIDI needs to ignore the delay before the end-of-song event. I'm not sure if that will fix it, though.

**Comment by Allofich** [comment]
I just tried and I don't have a looping issue with either of the two MIDIs mentioned in https://github.com/afritz1/OpenTESArena/issues/38 as having looping problems (OVERSNOW.XMI and SUNNYDAY.XMI)

Was this fixed somewhere along the way?

**Comment by afritz1** [comment]
It's been using SUNNYDAY.XFM instead of SUNNYDAY.XMI, since the .XFM version doesn't have any looping problems. I don't think WildMIDI has fixed the looping on their side yet.

**Comment by Allofich** [comment]
Yeah, I overlooked that. I'll test again explicitly using the .XMI files, and I'll also try the latest WildMIDI, just in case.

**Comment by afritz1** [comment]
See #38 for more details. I'm keeping this issue around until XMI looping for SUNNYDAY.XMI and OVERSNOW.XMI is properly handled by WildMIDI.
